### PR TITLE
Basic scaffold for managing authorised email domains

### PR DESCRIPTION
**Pull request by emileswarts**

### CODE REVIEW

File: app/controllers/admin/authorised_email_domains_controller.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+class Admin::AuthorisedEmailDomainsController < AdminController
+  def index
+    @authorised_email_domains = AuthorisedEmailDomain.all
```
Comment (georgeschena) [inline app/controllers/admin/authorised_email_domains_controller.rb]: Is this used to list all the allowed domains?

Comment (emileswarts) [reply app/controllers/admin/authorised_email_domains_controller.rb]: Yup

File: app/views/admin/authorised_email_domains/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+ <%= @authorised_email_domains.each do |custom_email_domain| %>
```
Comment (tommotaylor) [inline app/views/admin/authorised_email_domains/index.html.erb]: don't think you need this `=`

Comment (emileswarts) [reply app/views/admin/authorised_email_domains/index.html.erb]: Ah yes good catch

**Pull request merged**